electronics/_kicad_library/Polarity.pretty/Polarity_Center_Positive_2mm_SilkScreen.kicad_mod

```diff
@@ -0,0 +1,169 @@
+(module Polarity_Center_Positive_2mm_SilkScreen (layer F.Cu) (tedit 0)

```

Comment:
It's based on [Symbols.pretty](https://github.com/KiCad/Symbols.pretty) from the core repos; I can call it "Polarity_Marks.pretty" if that would help?